gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oidc/service/clientregistration/impl/DynamicClientRegistrationServiceImpl.java

```diff
@@ -65,6 +77,12 @@
 public class DynamicClientRegistrationServiceImpl implements DynamicClientRegistrationService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DynamicClientRegistrationServiceImpl.class);
+    public static final int FIVE_MINUTES_IN_SEC = 300;
+    public static final String FAPI_OPENBANKING_BRAZIL_DIRECTORY_JWKS_URI = "fapi.openbanking.brazil.directory.jwks_uri";
+    public static final String FAPI_OPENBANKING_BRAZIL_ACCESS_TOKEN_VALIDITY = "fapi.openbanking.brazil.access_token.validity";
+    public static final String FAPI_OPENBANKING_BRAZIL_REFRESH_TOKEN_VALIDITY = "fapi.openbanking.brazil.refresh_token.validity";
+    public static final String FAPI_OPENBANKING_BRAZIL_ID_TOKEN_VALIDITY = "fapi.openbanking.brazil.id_token.validity";
```

Comment:
I wonder if those settings can also be used for the generic DCR protocol 🤔 

Something like : 

```
openid:
  fapi:
     openbanking:
        brazil:
           directory:
              jwks_uri: 
  dcr:
    access_token:
      validity: 
    refresh_token:
      validity: 
    id_token:
      validity: 
```

WDYT ?

Reply:
yes, it will be better.

Anyway, this is a first step because a tab in the DCR settings page should be better, that will allow to configure these values per domain (and the values into the yaml file as default values)